projects/plugins/jetpack/functions.opengraph.php

```diff
@@ -533,3 +537,118 @@ function jetpack_og_get_description( $description = '', $data = null ) {
 
 	return $description;
 }
+
+/**
+ * Display a Fediverse actor Open Graph tag when the post author has a Mastodon connection.
+ *
+ * @see https://blog.joinmastodon.org/2024/07/highlighting-journalism-on-mastodon/
+ *
+ * @since $$next-version$$
+ *
+ * @param array $tags Current tags.
+ *
+ * @return array
+ */
+function jetpack_add_fediverse_creator_open_graph_tag( $tags ) {
+	/*
+	 * Let's not add any tags when the ActivityPub plugin already adds its own.
+	 * On WordPress.com simple, let's check if the plugin is active.
+	 * On self-hosted, let's just check if the class exists.
+	 */
+	$is_activitypub_active = function_exists( 'wpcom_activitypub_is_active' )
+		? wpcom_activitypub_is_active()
+		: class_exists( '\Activitypub\Integration\Opengraph' );
```

Comment:
We must be that verbose here because on WordPress.com Simple `\Activitypub\Integration\Opengraph` may be available while the ActivityPub flag hasn't been toggled for the site.